doc/source/how-to/index.rst

```diff
@@ -1,6 +1,5 @@
-#############################
-Guidelines and Best Practices
-#############################
+How-To
+######
```

Comment:
I don't know where to place this comment but... could you please rename images/Guidelines_chart.png to images_guidelines_chart.png. That way, all images have the same name style.